views/doc.list.html.twig

```diff
@@ -3,36 +3,50 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <section role="main">
+
+        <div id="searchbar" class="clearfix">
+            <input id="docsearch" type="text" placeholder="Type here to search documentation..." role="searchbox"
+                   title="Enter text here and submit to search the documentation"/>
+        </div>
+
+        <h1 role="sectionhead">📚 Book</h1>
+        <ul>
+            {% for filename, data in book if book is defined %}
+                {{ macros.renderFile(filename, data) }}
+            {% endfor %}
+        </ul>
+
+        <h1 role="sectionhead">📰 Articles</h1>
+        <ul>
+            {% for filename, data in articles if articles is defined %}
+                {{ macros.renderFile(filename, data) }}
+            {% endfor %}
+        </ul>
+
+        <h1 role="sectionhead">❓ FAQs</h1>
+        <ul>
+            {% for filename, data in faqs if faqs is defined %}
+                {{ macros.renderFile(filename, data) }}
+            {% endfor %}
+        </ul>
+
+        <h1 role="sectionhead">📄 API Docs</h1>
+        <p>If you need to work with composer as a library you may also want to check out the <a
+                    href="/apidoc/master/index.html"
+                    title="Go to the API Docs index"
+                    aria-label="Go to the API Docs index">API Docs</a>
+        </p>
+
+    </section>
 {% endblock %}
 
 {% macro renderFile(filename, data) %}
     <li>
-        <a href="{{ data.link }}">{{ filename }}</a>
-        {% if data.metadata is defined and data.metadata.tagline is defined %}<span class="tagline">{{ data.metadata.tagline }}</span>{% endif %}
+        <a href="{{ data.link }}" title="{{ filename|escape }}" aria-label="Link to: {{ filename|escape }}">
+            {{ filename }}
+        </a>
+        {% if data.metadata is defined and data.metadata.tagline is defined %}<span
+                class="tagline">{{ data.metadata.tagline }}</span>{% endif %}
```

Comment:
this line wrapping inside the HTML tag is quite weird.

Reply:
I did respect the 80 columns that were already used in all the repository. Although they're not mentioned explicitely, all files were like that. :/

Reply:
I'll put the tag on a new line! :)

Reply:
Given the amount of places were you added such weird wrapping, I would say that there was no such 80 column limit before your change.